.gitignore

```diff
@@ -33,3 +33,6 @@ lerna-debug.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+
+# Environment variables
+**/.env
```

Comment:
nit: Add end of line (EOL) character at the end of the file to conform to best practices.